.teamcity/Gradle_Check/configurations/FunctionalTest.kt

```diff
@@ -45,7 +45,7 @@ class FunctionalTest(
 
     applyTestDefaults(model, this, testTasks, notQuick = !testCoverage.isQuick, os = testCoverage.os,
         extraParameters = (
-            listOf(""""-PtestJavaHome=%${testCoverage.os.name.toLowerCase()}.${testCoverage.testJvmVersion}.${testCoverage.vendor}.64bit%"""") +
+            listOf(""""-PtestJavaHome=%${testCoverage.os}.${testCoverage.testJvmVersion}.${testCoverage.vendor}.64bit%"""") +
```

Comment:
```suggestion
            listOf(""""-PtestJavaHome=%${testCoverage.os.lowerCase()}.${testCoverage.testJvmVersion}.${testCoverage.vendor}.64bit%"""") +
```

Shouldn't this be ☝️ ?